pkg/vm/vm_test.go

```diff
@@ -2031,6 +2031,52 @@ func TestPACKGood(t *testing.T) {
 	assert.Equal(t, int64(1), vm.estack.Peek(1).BigInt().Int64())
 }
 
+func TestPACK_UNPACK_MaxSize(t *testing.T) {
```

Comment:
Happily, reference counter is tied to stack and updated upon every pop/push, so there are no bugs here :)